### PR TITLE
Re-add TMO

### LINKED ISSUE #2438: [Bug] TuMangaOnline : Change Domain
The TuMangaOnline site changed its domain url to: https://lectortmo.com/

### PR DESCRIPTION
**Pull request by happywillow0**
Closes #2438

Apart from the server migration, the chapter authentication method hasn't changed in the last month. 
The fake images also was removed. 

This re-adds, TMO. Let me know if we should continue the version at where we left off or we can start over.

### CODE REVIEW

**Comment by MerradOklom** [comment]
Maybe this should stay on hold for a while, since they changed their domain I have seen several reports on their forum about broken images, images not loading, etc. I'm afraid people will report it as an extension problem.

**Comment by happywillow0** [comment]
They fixed the ssl certificate issue on their image server (causing the broken images / images not loading) which is why I went ahead and opened up the pull.

**Comment by happywillow0** [comment]
Nevermind, I'll close this for now why we are still holding. We'll see what news they put out for updates.  Feel free to ping me if and when I should open this again.

**Comment by MerradOklom** [comment]
Yep, their site is a bit of a mess right now. Even some scanlators are unable to access their groups to upload manga. I'll let you know if there is any good news to share.

**Comment by MerradOklom** [comment]
Looks like the site has been stabilized, I haven't seen reports of broken images in more than a week.
They also added new filters (by quantity of chapters and Scanlation statuses), if you didn't include them in this pr maybe I can open a feature request after it has been merged.

**Comment by happywillow0** [comment]
Although the servers are working, they seem still slow for me and images take time to load. 

Currently downloading manga from them may be a good way to read.

File: src/es/tumangaonline/build.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    appName = 'Tachiyomi: TuMangaOnline'
+    pkgNameSuffix = 'es.tumangaonline'
+    extClass = '.TuMangaOnline'
+    extVersionCode = 1
```
Comment (arkon) [inline src/es/tumangaonline/build.gradle:8]: Might want to just increase it beyond what was there before so whoever might still have it installed will see it as an update.

Comment (happywillow0) [reply src/es/tumangaonline/build.gradle]: Increased it based on the removal commit 2196
Done

**Pull request merged**